plugin/src/main/java/io/jenkins/plugins/analysis/core/model/SourcePrinter.java

```diff
@@ -199,6 +199,13 @@ private String selectLanguageClass(final Issue issue) {
                 return "language-clike"; // Best effort for unknown extensions
         }
     }
+    private String asMarkedCode(final StringBuilder text, final Issue issue, final String... classes) {
+        final StringBuilder marked = columnMarker.markColumns(text.toString(), issue.getColumnStart(), issue.getColumnEnd());
+        final String sanitized = SANITIZER.render(StringEscapeUtils.escapeHtml4(marked.toString()));
+        final String markerReplaced = columnMarker.replacePlaceHolderWithHtmlTag(sanitized);
+        final UnescapedText unescapedText = new UnescapedText(markerReplaced);
+        return code().withClasses(classes).with(unescapedText).render();
```

Comment:
I wonder if we should skip the marking, if the issue is for multiple lines. In this case `start` and `end` will be on different lines. But for a multi-line mark the extra highlighting does not make much sense so skipping might be the best option.

Reply:
I am not sure, but I would assume in case of a multiline range, issue.GetColumnStart() will return `0`. In this case `columnMarker.markColumns` does nothing and gives back the unmodified input.

Reply:
This is not true for PMD: https://ci.jenkins.io/job/Plugins/job/analysis-model/job/master/1324/pmd/source.53a605b6-b7dd-4ec8-81fe-27af1f73715c/#32

This warning is:
```
<violation beginline="32" endline="404" begincolumn="8" endcolumn="1" rule="GodClass" ruleset="Design" package="edu.hm.hafner.analysis" class="ModuleDetector" externalInfoUrl="https://pmd.github.io/pmd-6.40.0/pmd_rules_java_design.html#godclass" priority="3">
Possible God Class (WMC=48, ATFD=23, TCC=10.526%)
</violation>
```